antslib/etc/ants.cfg

```diff
@@ -4,6 +4,7 @@ branch = master
 ssh_key = /etc/ants/id_ants
 destination = ~root/.ants_playbook
 inventory_script = inventory_default
+ansible_callback_whitelist = logstash
```

Comment:
I think we shouldn't set logstash as a default callback plugin. Could we leave it empty like `tags` for example ad add an entry about supported callback plugins to the `README`?

Reply:
Yep sounds right to me.